Formula/arangodb.rb

```diff
@@ -73,7 +67,10 @@ def install
       -DUSE_GOOGLE_TESTS=Off
       -DCMAKE_INSTALL_LOCALSTATEDIR=#{var}
     ]
-    args << "-DTARGET_ARCHITECTURE=nehalem" if build.bottle? && Hardware::CPU.intel?
+    llvm = Formula["llvm@13"] if OS.mac?
+    args << "-DCMAKE_C_COMPILER=#{llvm.opt_bin}/clang" if OS.mac?
+    args << "-DCMAKE_CXX_COMPILER=#{llvm.opt_bin}/clang++" if OS.mac?
+    args << "-DTARGET_ARCHITECTURE=sandy-bridge" if build.bottle? && Hardware::CPU.intel?
```

Comment:
```suggestion
    args << "-DTARGET_ARCHITECTURE=#{Hardware.oldest_cpu}" if build.bottle? && Hardware::CPU.intel?
```
Sandy Bridge is too new a target.

Reply:
Ok, we will use oldest_cpu